### PR TITLE
Implemented login form

### PR DESCRIPTION
**Pull request by jaduff**
Implemented basic User model

### CODE REVIEW

File: src/LabLog/Views/Teacher/Index.cshtml [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-<p>This page is the admin landing page.</p>
-<p>It will contain a list of rooms, as well as
-    the options to create a new room. Room editing and deleting will be done from
-    within each Room's view page.</p>
\ No newline at end of file
+  <div class="wrapper">
+    <form class="form-signin" action="/Teacher" method="POST">
+      <h2 class="form-signin-heading">Please login</h2>
+      <input type="text" class="form-control" name="username" placeholder="E-Number" required="True" autofocus="" />
+      <input type="password" class="form-control" name="password" placeholder="Password" required="True"/>
```
Comment (simonjduff) [inline src/LabLog/Views/Teacher/Index.cshtml]: there are special generator functions you can use to help with these. [Tag Helpers](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/intro)

File: src/LabLog/Views/Teacher/Index.cshtml [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
\ No newline at end of file
+  <div class="wrapper">
+    <form class="form-signin" action="/Teacher" method="POST">
+      <h2 class="form-signin-heading">Please login</h2>
+      <input type="text" class="form-control" name="username" placeholder="E-Number" required="True" autofocus="" />
+      <input type="password" class="form-control" name="password" placeholder="Password" required="True"/>      
+      <label class="checkbox">
+        <input type="checkbox" value="remember-me" id="rememberMe" name="rememberMe"> Remember me
+      </label>
+      <button class="btn btn-lg btn-primary btn-block" type="submit">Login</button>
```
Comment (simonjduff) [inline src/LabLog/Views/Teacher/Index.cshtml]: Definitely use the tag helpers with the button, it'll link it to a particular controller and action and allow for easy refactoring

File: src/LabLog/Views/Teacher/Index.cshtml [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
-<p>It will contain a list of rooms, as well as
-    the options to create a new room. Room editing and deleting will be done from
-    within each Room's view page.</p>
\ No newline at end of file
+  <div class="wrapper">
+    <form class="form-signin" action="/Teacher" method="POST">
+      <h2 class="form-signin-heading">Please login</h2>
+      <input type="text" class="form-control" name="username" placeholder="E-Number" required="True" autofocus="" />
+      <input type="password" class="form-control" name="password" placeholder="Password" required="True"/>      
+      <label class="checkbox">
```
Comment (simonjduff) [inline src/LabLog/Views/Teacher/Index.cshtml]: when using a label element, set the "for" attribute, which will bind a mouseclick to the text of the label to the form element it's associated with. There's also a tag helper for these too

Comment (jaduff) [reply src/LabLog/Views/Teacher/Index.cshtml]: Whoops, yeah I those for the text inputs, but then deleted the label entirely in favour of a placeholder. Forgot about the radio box.

Comment (jaduff) [reply src/LabLog/Views/Teacher/Index.cshtml]: Can't use the asp-for tag helper for the RememberMe checkbox, as there isn't a matching entry in the UserModel. Should there be?

File: src/LabLog/Views/Teacher/Index.cshtml (hunk 1 above)
Comment (jaduff) [reply src/LabLog/Views/Teacher/Index.cshtml]: Does asp-for make sense in an input field?
`<input asp-for="Email" class="form-control" />`
Example taken from your link.
What is the purpose of asp-for in an input field? It already binds mouseclick to itself.

**Pull request closed without merging**